game/gothic.h

```diff
@@ -86,6 +86,9 @@ class Gothic final {
     bool         isMarvinEnabled() const;
     void         setMarvinEnabled(bool m);
 
+    bool         isGodMode() const {return godMode; }
+    void         setGodMode() { godMode = !godMode; }
```

Comment:
`set` function take no arguments, therefore it's more of `toogleGodMode`. 
Can you please change this to `setGodMode(bool)`?